src/OnboardingSPA/pages/Steps/DesignThemeStyles/Preview/index.js

```diff
@@ -199,7 +199,7 @@ const StepDesignThemeStylesPreview = () => {
 					<span className="theme-styles-preview__title-bar__browser__dot"></span>
 				</div>
 			</div>
-			<div className="theme-styles-preview__live-preview-container">
+			<div className="theme-styles-preview__live-preview-container live-preview-container">
```

Comment:
Again Is there a reason we have an extra `live-preview-container` here? Why not use `theme-styles-preview__live-preview-container`

Reply:
The same reason cause live-preview-container is global and a lot of CSS is already duplicated and there should be some CSS which all Pages Live Preview is gonna use.

Reply:
Removed It.